core/src/main/java/com/linecorp/armeria/common/util/AsyncCloseable.java

```diff
@@ -22,12 +22,24 @@
  * method releases the resources asynchronously, returning the {@link CompletableFuture} which is completed
```

Comment:
need to change the doc a little bit?
Unlike {@link AutoCloseable}, -> In addition to the funcationality of closing synchronously with {@link AutoCloseable},